.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: ruby
+sudo: false
 before_install:
   - gem install bundler
 rvm:
-  - 1.9.3
-  - 2.0.0
+  - 2.2.2
 gemfile:
   - Gemfile
   - gemfiles/Gemfile-edge
```

Comment:
also, edge is in `allow-failures` group
so a Gemfile-5-0-stable file should really be added